main.py

```diff
@@ -1,10 +1,36 @@
-"""This file will use all of our functions and run the progam"""
+"""This file will use all of our functions and run the program"""
 
+import time
+import typer
+
+# Import from configator
 from configator import create_actions_config
 from configator import create_gatorgradle_yml
 from configator import generate_build_gradle
 
-if __name__ == '__main__':
+app = typer.Typer()
+@app.command()
+def main(
+    name: str = typer.Option("configator-generated"),
+    brk: str = typer.Option("true"),
+    fastfail: str = typer.Option("false"),
+    ind: int = typer.Option(2),
+    vers: str = typer.Option("v1.1.0"),
```

Comment:
Instead of defaulting to `v1.1.0`, default to `master`. This way, the most recent version of GatorGrader will be used (even if it is updated after you publish this release).